configure.py

```diff
@@ -1893,7 +1892,7 @@ def configure_abseil(build_dir, mode, mode_config):
               pool = console
               description = TEST {mode}
             rule rust_lib.{mode}
-              command = CARGO_HOME=build/{mode}/rust/.cargo cargo build --release --manifest-path=rust/Cargo.toml --target-dir=build/{mode}/rust -p ${{pkg}}
+              command = CARGO_HOME=build/{mode}/.cargo RUSTFLAGS="--cfg=rustix_use_libc" cargo build --manifest-path=rust/Cargo.toml --target-dir=build/{mode} --profile=rust-{mode}
```

Comment:
This `RUSTFLAGS` value fixes a weird issue where `rustix` - a rust crate that wasmtime depends on - returns 0 as the system page size when using `linux_raw` instead of `libc` as the backend for systemcalls

Reply:
In case it helps, I believe this problem is fixed by https://github.com/bytecodealliance/rustix/pull/385, which is included in rustix 0.35.9.

Reply:
Thanks for the info. I updated the dependencies and confirmed that everything works now without this flag